src/app/views/App.tsx

```diff
@@ -127,7 +127,13 @@ class App extends Component<IAppProps, IAppState> {
   }
 
   private hashDecode(requestBody: string): string {
-    return requestBody ? atob(requestBody) : '';
+    const decodedBody = atob(requestBody);
+
+    if (decodedBody === 'undefined') {
```

Comment:
Out of curiosity, is there a specific case where this would occur?

From [this](https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/atob), I can only tell that `atob` will throw if the conversion fails.

Reply:
The throw exception occurs for non valid base64. This handles the case where a falsey value is  base64 encoded.

Reply:
Thanks so much for explaining that. 😄